src/current/v24.1/show-default-session-variables-for-role.md

```diff
@@ -0,0 +1,157 @@
+---
+title: SHOW DEFAULT SESSION VARIABLES FOR ROLE
+summary: The SHOW DEFAULT SESSION VARIABLES FOR ROLE statement lists the default values for session variables applied to a given user.
+keywords: reflection
+toc: true
+docs_area: reference.sql
+---
+
+The `SHOW DEFAULT SESSION VARIABLES FOR ROLE` [statement]({% link {{ page.version.version }}/sql-statements.md %}) lists the values for updated [session variables]({% link {{ page.version.version }}/set-vars.md %}) that are applied to a given [user/role]({% link {{ page.version.version }}/security-reference/authorization.md %}#roles).
+
+It only shows the values of session variables that are changed from the defaults. When no session variables have been changed from the defaults for a given role, the statement [returns no values](#output-when-no-session-variables-have-been-changed).
+
+## Synopsis
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/show_default_session_variables_for_role.html %}
```

Comment:
Dang you're right!

this appears to be what they are called in the SQL parser: https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/parser/sql.y#L1038 

in the past we had a method for changing these diagrams locally but now they are automatically generated by an external process.  I think I can tweak it with a PR against `cockroachdb/cockroach/pkg/cmd/docgen/diagrams.go`

Have filed a separate issue to address this since it will take some time to get a PR into CockroachDB: https://cockroachlabs.atlassian.net/browse/DOC-10114